auacm/app/modules/__init__.py

```diff
@@ -0,0 +1,8 @@
+'''Contains our backend global variables'''
+from flask import Flask
+
+# Create the God-objects to be used everywhere
+APP = Flask(__name__, template_folder='../templates',\
```

Comment:
`APP` should be lowercase here, as it's not a constant, related to above comment.